tasks/utils.py

```diff
@@ -306,21 +306,21 @@ def download(url, destination=None, options={}):
             logging.info("Downloading: %s" % url)
 
             if postdata:
-                response = scraper.urlopen(url, 'POST', postdata, **urlopen_kwargs)
+                response.post = scraper.urlopen(url, postdata, **urlopen_kwargs)
```

Comment:
This line doesn't look right.

Reply:
Thanks for the review!

Ah yes mb let me fix it rq.

Reply:
@JoshData fixed